Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -1699,7 +1699,8 @@ - (void)collectionView:(JSQMessagesCollectionView *)collectionView
                                     NSIndexPath *indexPathI = [NSIndexPath indexPathForRow:i inSection:0];
                                     id<OWSMessageData> message = [self messageAtIndexPath:indexPathI];
 
-                                    if (message.messageType == TSIncomingMessageAdapter && message.isMediaMessage) {
+                                    if (message.messageType == TSIncomingMessageAdapter && message.isMediaMessage &&
+                                        [[message media] isKindOfClass:[TSVideoAttachmentAdapter class]]) {
```

Comment:
When you start playing any video or audio file, we want to stop playback of any other video or audio files.  This logic was also trying to stop playback of images, which caused a crash. I don't understand how this ever worked.